packages/sanity/src/core/form/inputs/PortableText/text/TextBlock.tsx

```diff
@@ -79,6 +80,12 @@ export function TextBlock(props: TextBlockProps) {
   const memberItem = usePortableTextMemberItem(pathToString(path))
   const editor = usePortableTextEditor()
 
+  const presence = useChildPresence(path, true)
+  const rootPresence = useMemo(
```

Comment:
❓`rootPresence` is an unused, should it be used?

Reply:
Thank you for spotting this @binoy14 

I have updated the code slightly (amened last commit), please refresh.